tests/test_analysis.py

```diff
@@ -106,7 +106,19 @@ def test_get_pca_dimensions(self):
         np.testing.assert_allclose(VAF, single_dim_VAF, atol=1e-7)
         self.assertAlmostEqual(num_dims, single_num_dims)
 
-        
+class CalcTests(unittest.TestCase):
+
+    def test_calc_rms(self):
+        # sanity check
+        signal = np.array([1])
+        rms = aopy.analysis.calc_rms(signal)
+        self.assertEqual(rms, 0)
+
+        # check dimensions are preserved
+        signal = np.array([[0.5, -0.5], [1., -1.]]).T
+        rms = aopy.analysis.calc_rms(signal)
+        self.assertCountEqual(rms, np.array([0.5, 1.]))
```

Comment:
If I understand correctly this test ensures the two sequences contain the same elements regardless of order. I think it would be better to use an order preserving check like np.testing.assert_allclose